src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -23,21 +23,8 @@
     ONE_INPUT_TYPE_ONLY_ERR,
     INVALID_REQUEST_ERR,
     MISSING_REQUEST_TYPE_ERR,
-    BATCH_ID,
-    JOB_ID,
-    JOB_ID_LIST,
-    CELL_ID_LIST,
-    CANCEL,
-    CELL_JOB_STATUS,
-    ERROR,
-    INFO,
-    LOGS,
-    NEW,
-    RETRY,
-    START_UPDATE,
-    STATUS,
-    STATUS_ALL,
-    STOP_UPDATE,
+    MESSAGE_TYPE,
+    PARAM
```

Comment:
all changes in this file are from switching out message type constants for `MESSAGE_TYPE["..."]`